docs/development/issue_labeling.md

```diff
@@ -0,0 +1,221 @@
+
+We try to keep issues well-classified by labeling them appropriately.
+Only members/collaborators with the required rights can apply labels.
+
+## Basic knowledge about Renovate
+
+You should know about platforms, package managers, datasources and versioning to label issues effectively.
+
+- To learn about platforms, read the [Renovate docs on Platforms](https://docs.renovatebot.com/modules/platform/).
+- To learn about managers, read the [Renovate docs on Managers](https://docs.renovatebot.com/modules/manager/).
+- To learn about datasources, read the [Renovate docs on Datasources](https://docs.renovatebot.com/modules/datasource/).
+- To learn more about versioning, read the [Renovate docs on Versioning](https://docs.renovatebot.com/modules/versioning/).
+
+## Label categories
+
+### Priority
+
+<details>
+    <summary>Priority</summary>
+
+    pri0-blocker
+    pri1-critical
+    pri2-important
+    pri3-normal
+    pri4-low
+    wontfix
+
+</details>
+
+Use these to assign a priority level to an issue.
+
+QUESTION: Who gets to decide the priority level? @rarkins and maintainers, or can a triaging member select a priority level?
+
+QUESTION: Maybe we should copy paste the priority descriptions from the labels into this doc?
+
+### Type of issue
+
+<details>
+    <summary>Type of issue</summary>
+
+    bug
+    docs
+    duplicate
+    feature
+    refactor
+    breaking
+
+</details>
+
+Use these to label the type of issue.
+For example, use `bug` to label a bug type issue, and use `feature` for feature requests.
+Only use `refactor` for code changes, don't use `refactor` for documentation type changes.
+
+QUESTION: Do we use breaking for "breaking" changes in a PR, or as a "the house is on fire" marker for issues, or both?
+Use the `breaking` label for pull requests that have breaking changes.
+Use the `breaking` label for when a bug is realy major.
+
+QUESTION: Do we even need a `duplicate` label, there's **mark as duplicate** functionality in GitHub nowadays...
+
+### Housekeeping
+
+<details>
+    <summary>Housekeeping</summary>
+
+    good first issue
+    help wanted
+    needs reproduction
+    reproduced
+
+</details>
+
+Add a label `good first issue` to issues that are small, easy to fix, and do-able for a newcomer.
+
+Add a label `needs reproduction` if the issue can only be fixed once there's a reproduction.
+Add the label `reproduced` once there is a public reproduction.
+
+QUESTION: When/who should use the `help wanted` label?
+
+### Miscellaneous
+
+<details>
+    <summary>Miscellaneous</summary>
+
+    gitfs
+    hacktoberfest
+    hacktoberfest-accepted
+    internal
+    ready
+    self hosted
+
+</details>
+
+QUESTION: What is the `gitfs` label about?
+QUESTION: Should we document the `hacktoberfest` labels?
+QUESTION: What is the `internal` label about?
+QUESTION: Do we even use the `ready` label now Renovate has a project board?
+QUESTION: I think `self hosted` belongs in the `platform:` type label section instead???
```

Comment:
Any of the platforms can be self-hosted, while both GitHub and GitLab can be hosted. I use this label to identify when an issue is applicable only to users who self-administer their own bot.